src/Chapter1.hs

```diff
@@ -568,7 +578,9 @@ True
 >>> isVowel 'x'
 False
 -}
-isVowel c = error "isVowel: not implemented!"
+
+isVowel :: Char -> Bool
+isVowel c = c == 'a' || c == 'e' || c == 'i' || c == 'o' || c == 'u'
```

Comment:
The description asked to use guards but it didn't make sense to me here since the conditional already returns a `Bool`. Is there a reason why guards were recommended here?